sql/core/src/main/scala/org/apache/spark/sql/parquet/newParquet.scala

```diff
@@ -545,13 +578,16 @@ object ParquetRelation2 {
   // Whether we should merge schemas collected from all Parquet part-files.
   val MERGE_SCHEMA = "mergeSchema"
 
-  // Hive Metastore schema, passed in when the Parquet relation is converted from Metastore
+  // Hive Metastore schema, passed in when the Parquet relation is converted from Metastore.
   val METASTORE_SCHEMA = "metastoreSchema"
 
-  // Default partition name to use when the partition column value is null or empty string
+  // Schema of partition keys for partitioned Hive Metastore Parquet tables.
+  val METASTORE_PARTITION_KEYS_SCHEMA = "metastorePartitionKeysSchema"
```

Comment:
Ah, forgot to remove this. It's not used any more since we construct `PartitionSpec` manually and pass it to `ParquetRelation2` when converting Metastore Parquet tables.